## üëã Hi there 
My name is Nilesh Domah and I am currently pursuing a Bioinformatics/Data Scientist position. I got my Master of Science in Biomedical Informatics (with a Bioinformatics concentration) from the University of Chicago and a Bachelor of Arts in Biology from the University of Minnesota - Twin Cities. I have ~2 years of experience working as a data analyst. 

## üìä Projects
**üß¨ Bioinformatics**:
- [Machine Learning](https://github.com/ndomah001/Machine-Learning)
  - Developed and implemented machine learning models to classify E. Coli DNA sequences, achieving up to 93% accuracy in promoter gene identification.
  - Evaluated 10 different algorithms using cross-validation, demonstrating proficiency in Random Forest, Support Vector Machines, and K-Nearest Neighbors.
  - Analyzed model performance using precision, recall, and F1-scores to identify optimal classifiers for genomic data analysis
- [Single-Cell RNA-seq Analysis](https://github.com/ndomah001/scRNA-Seq-Analysis)
  - Performed comprehensive single-cell RNA-seq analysis on ~20,000 NSCLC cells using Seurat in R, including quality control, dimensionality reduction, and clustering.
  - Applied advanced data visualization techniques (PCA, UMAP) to reveal 13 distinct cell populations within the NSCLC tumor microenvironment.
  - Conducted in-depth statistical analysis of gene expression patterns, identifying key variable genes and principal components driving cellular heterogeneity in NSCLC
- [Weighted Gene Co-expression Network Analysis](https://github.com/ndomah001/Weighted-Gene-Co-expression-Network-Analysis)
  - Conducted Weighted Gene Co-expression Network Analysis (WGCNA) on RNA-seq data from 34 subjects to identify gene modules associated with COVID-19 and disease severity.
  - Applied advanced bioinformatics techniques including hierarchical clustering, Principal Component Analysis, and soft-thresholding power optimization for robust network construction.
  - Identified key gene modules significantly correlated with COVID-19 status and severity, providing insights into potential disease mechanisms and therapeutic targets
- [Differential Gene Expression Analysis](https://github.com/ndomah001/Differential-Gene-Expression)
  - Conducted differential gene expression analysis using DESeq2 on RNA-seq data from TGF-Œ≤ treated and control samples, identifying key genes involved in epithelial-to-mesenchymal transition (EMT).
  - Applied advanced statistical techniques including dispersion estimation and multiple testing correction to ensure robust identification of significantly differentially expressed genes.
  - Utilized data visualization methods (PCA, volcano plots, heatmaps) to effectively communicate complex genomic data, revealing distinct expression profiles between TGF-Œ≤ treated and control samples

## ‚öíÔ∏è Skills:
- **Programming Languages**: R, Python, Bash Scripting, SQL
  - **Python Packages**: Biopython, Pandas, NumPy, SciPy, Matplotlib, Seaborn, Scikit-learn, PyTorch, TensorFlow, PySAM, HTSeq, DendroPy
  - **R Packages**: Bioconductor, DESeq2, edgeR, limma, WGCNA, ggplot2, GenomicRanges, SummarizedExperiment, ComplexHeatmap, gplots, ape
- **Pipelines**: Nextflow, Snakemake, Galaxy
- **Developer Tools**: Jupyter Notebook, R Shiny App, VSCode, Git, Streamlit
- **Cloud**: Azure, AWS, Google Cloud Platform


## üîó Links:
- LinkedIn:
- Portfolio Website:
<!--
**ndomah001/ndomah001** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
